<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sphinx Patcher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_xbox">
       <attribute name="title">
        <string>Xbox</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_shadows">
          <property name="title">
           <string>Shadows</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="radioButton_shadows_projected">
             <property name="text">
              <string>Projected (invisible on Xbox 360)</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_shadows_simple">
             <property name="text">
              <string>Simple</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_shadows_none">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>None</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_aspect">
          <property name="title">
           <string>Aspect ratio</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioButton_aspect_save">
             <property name="text">
              <string>Save file</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_aspect_system">
             <property name="text">
              <string>System (recommended)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_aspect_normal">
             <property name="text">
              <string>Force 4:3 (fixes 1024x768)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_aspect_wide">
             <property name="text">
              <string>Force Widescreen/custom</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_widescreen">
          <property name="title">
           <string>Widescreen</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBox_widescreen_aspect">
             <property name="text">
              <string>Custom aspect ratio:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_ratio">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>16:10</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBox_widescreen_2dwidth">
             <property name="text">
              <string>Custom 2D width:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_width">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>1.0</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_gamecube">
       <attribute name="title">
        <string>Nintendo GameCube</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_ps2">
       <attribute name="title">
        <string>PlayStation 2</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_pc">
       <attribute name="title">
        <string>PC</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_switch">
       <attribute name="title">
        <string>Nintendo Switch</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_switch">
          <property name="text">
           <string>Click &quot;Patch&quot; to download the IPSwitch patches.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_patch">
      <property name="text">
       <string>Patch</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
